### DISCUSSION TITLE
How do I get the generated PR address on each run? (self-hosted)

### DISCUSSION THREAD

**Discussion by songkeys** [renovatebot/renovate#8955] [Request Help]
I can't see anything related to url output to the console. Is it possible to give these info? Thanks.

**Comment by viceice** [comment]
`LOG_LEVLEL=debug`

**Reply by songkeys** [reply]
I know. But the output information is so verbose. And at the end of the logs, it's hard to see if a PR is created successfully or not due to some sorts of issues.

I'm expecting some log messages like:

```
A PR has been created at [url].

// or

No PR has been created because of...
```

**Reply by rarkins** [reply]
You can search for "PR created" and "PR updated" INFO messages to start with. Is the end purpose for you to channel this data into some other system?

**Reply by songkeys** [reply]
Thank you. Yes! I need this information to notify users about the status.

So currently I'm filtering `"PR created"` and parsing the ID out to concat urls. This is doable but there're some other cases I think? Like "not created", and some issues are created in some cases, etc. I'm not so sure about the format and cases. It'd be good If there's a guide or description about this on docs.

But I think hooks like "postPrCreated" are best for renovate.
